src/main/resources/selects.js

```diff
@@ -0,0 +1,27 @@
+function checkState(select) {
+  if (select.tagName.toLowerCase() !== 'select') return {'nonSelect': 'Cannot select option from a non-select element'}
+  if (select.disabled) return {'disabledSelect': 'Cannot select option in a disabled select'}
+  return null
+}
+
+function getSelectedOptionsString(select) {
+  return Array.from(select.options).map(option => option.selected).join(",")
+}
+
+function optionByValue(select, requestedValue) {
+  return Array.from(select.options).find(option => option.value === requestedValue)
+}
+
+function applyChanges(select, changes) {
+  let previousSelectedOptions = getSelectedOptionsString(select)
+
+  select.focus()
+  changes(select)
+
+  select.dispatchEvent(new Event('click'));
```

Comment:
It will be nice to restore event bubbling as it was [before](https://github.com/selenide/selenide/blob/v7.4.0/src/main/resources/select-options-by-value.js#L34)

Reply:
@BorisOsipov Good point. Fixed.